packages/server/middleware/auth.js

```diff
@@ -2,8 +2,9 @@ const firebase = require('firebase-admin');
 
 async function authMiddleware(req, res, next) {
   try {
+    // The token should only ever be sent in a header.
     req.user = await firebase.auth()
-      .verifyIdToken(req.query.token || req.headers.token || req.body.token);
```

Comment:
Can we just remove req.query? I think it is fine for us to be getting the token from the body.

Reply:
I think that GET request will put the token in the query and we want to make sure the middleware handles all the cases. Ideally, all requests should put it in the header but I don't think our code abides by that